x/lend/keeper/asset.go

```diff
@@ -135,3 +135,15 @@ func (k *Keeper) GetAssetForDenom(ctx sdk.Context, denom string) (asset types.As
 
 	return k.GetAsset(ctx, id.GetValue())
 }
+
+func (k *Keeper) GetBalancesForModule(ctx sdk.Context, module string) (balances string) {
+	if module != "osmo" && module != "cmdx" && module != "atom" {
+		return "0"
+	}
+	moduleBalances := k.bank.GetAllBalances(ctx, k.account.GetModuleAddress(module))
+	result := moduleBalances.String()
+	if result == "" {
+		return "0"
```

Comment:
balance type as string ? any reason ?

Reply:
yes because there can be multiple denoms per module address and format would be {amount}{denom},{amount}{denom}

Reply:
so its an array of maps ? do you think string is better to handle here ?